kubel-evil.el

```diff
@@ -53,6 +53,13 @@
 (require 'kubel)
 
 ;;; Code:
+(define-minor-mode kubel-evil-mode
+  "Brings evil keybindings to kubel"
+  :lighter " kubel-evil"
+  :keymap (make-sparse-keymap)
+  :group 'kubel-evil)
+
+(add-hook 'kubel-mode-hook 'kubel-evil-mode)
```

Comment:
does this mean that `kubel-evil-mode` will be enabled by default everytime `kubel-mode` is enabled?

Reply:
Yes, but only if kubel-evil would be present. By default it's excluded if kubel is installed via melpa. So I thought if someone decides to install kubel-evil as well it should be loaded automatically. What do you think? Should I remove the hook again?

Reply:
I think this is the same behavior as before the package contained a minor mode.  But now the user is able to turn off `kubel-evil-mode` within the buffer, which is a good thing.

Reply:
Makes sense! Thanks